src/hooks/Beamer/useBeamerNps.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react'
+
+import { BEAMER_ID } from '@/config/constants'
+
+const LS_KEY = `_BEAMER_NPS_LAST_SHOWN_${BEAMER_ID}`
+
+export const useBeamerNps = (): void => {
+  useEffect(() => {
+    if (typeof window === 'undefined' || !window.Beamer) {
+      return
+    }
+
+    const lastShown = window.Beamer.getCookie(LS_KEY)
+
+    if (!lastShown) {
+      window.Beamer.forceShowNPS()
```

Comment:
Can you link to Beamer docs about this fn? Why do we "force" it?

Reply:
Have spoken with Beamer regarding this, as well as asked some other questions. I'll make this a draft for now and include what I hear back.

Reply:
As per our discussion, I've added comments to address this.